docs/ar/week03/03.md

```diff
@@ -0,0 +1,27 @@
+| lang-ref | lang | title | translator |
+| ------ | ------ | ------ | ------ |
+| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |
+
+<div dir="rtl">

```

Comment:
Remove this.
My script is reading the `MD` file and serving a `HTML`.
We'll fix alignment in the `HTML` page.